docs/text_handling.md

```diff
@@ -4,6 +4,19 @@ OpenVINO Model Server can now greatly simplify writing the applications with Nat
 
 We addressed both the situation when the original model requires tokens on input or output and there is added support for models with embedded tokenization layer. Below are demonstrated use cases with a simple client application sending and receiving text in a string format. Whole complexity of the text conversion is fully delegated to the remote serving endpoint.
 
+## Accepting strings by serving Python script
+
+When the model server is configured to serve python script (via MediaPipe Graph with PythonCalculator), it is possible to communicate with arbitrary format conforming Python's buffer protocol including string or list of strings. Refer to full [Python integration documentation](python_support/quickstart.md) or end-to-end [LLM text generation demo](../demos/python_demos/llm_text_generation/README.md).
```

Comment:
1. Please use a full name: PythonExecutorCalculator.
2. Mentioning buffer protocol seems like too much of a detail on this level. Maybe it would be better to just say that it's possible to work with encoded strings? 
3. [Python integration documentation] - maybe let's stick with Python "execution" rather than "integration" (we don't use "Python integration" in other parts of the documentation)
4. You are linking to quickstart not to the full documentation.

Reply:
Done